### ISSUE TITLE
Style Error in React

### ISSUE THREAD

**Issue by ChaseWindYoungs** [snettah/react-ztext#3]
In Vue-ztext, i can find z-layer class name. but in react, it seems like meaningless style names

**Comment by snettah** [comment]
Hello, maybe the `layers' props ?

``` ts
interface Props {
  depth: string
  direction: string
  event: string
  eventRotation: string
  eventDirection: string
  fade: boolean
  layers: number
  perspective: string
  children: React.ReactNode
  style?: React.CSSProperties
}
```

**Comment by ChaseWindYoungs** [comment]
> Hello, maybe the `layers` props ?
> 
> ```ts
> interface Props {
>   depth: string
>   direction: string
>   event: string
>   eventRotation: string
>   eventDirection: string
>   fade: boolean
>   layers: number
>   perspective: string
>   children: React.ReactNode
>   style?: React.CSSProperties
> }
> ```

Seems so, why not use z-layer in react, I can just make some css changes using this style name

**Comment by raymolla7** [comment]
Yeah I am having same issue. Just meaningless random class names in React version:

![image](https://github.com/user-attachments/assets/0ae7e1f1-6043-43d5-ab68-89175bf2c336)

But in the regular version it's supposed to look like this:

![image](https://github.com/user-attachments/assets/75fe555a-5cd6-4f2a-bceb-200172ca45b1)

Having the regular classnames is the intended way to do styling as per the site:

![image](https://github.com/user-attachments/assets/fc7ee7d5-330f-4d55-80a9-a6c698357bc7)
